scripts/main.js

```diff
@@ -52,11 +52,11 @@ module.exports = (robot) => {
         }
     })
     robot.error((err, res) => {
-        console.log(err)
-        robot.logger.error("DOES NOT COMPUTE")
+        console.log(err.message)
+        robot.logger.error(err.message)
```

Comment:
@zelima  I will add you to our new channel for bot testing, and i will test there